test/test.ts

```diff
@@ -854,58 +847,60 @@ describe('gcs-resumable-upload', () => {
     it('should set userProject', (done) => {
       up.authClient = {
         authorizeRequest(reqOpts: RequestOptions) {
-          assert.deepEqual(reqOpts.qs, {userProject: USER_PROJECT});
+          assert.deepEqual(reqOpts.params, {userProject: USER_PROJECT});
           done();
         }
       };
       up.getRequestStream(REQ_OPTS);
     });
 
     it('should destroy the stream if an error occurred', (done) => {
-      const error = new Error(':(');
+      const error = 'ಠ_ಠ';
       up.destroy = (err: Error) => {
-        assert(err.message.indexOf(error.message) > -1);
+        assert(err.message.indexOf(error) > -1);
         done();
       };
-      up.authClient = {
-        authorizeRequest(
-            reqOpts: RequestOptions, callback: AuthorizeRequestCallback) {
-          callback(error, null!);
-        }
-      };
-      up.getRequestStream(REQ_OPTS);
+      const scope = mockAuthorizeRequest(500, error);
+      up.getRequestStream(REQ_OPTS, (stream: Request) => {
+        scope.done();
+      });
     });
 
     it('should make the correct request', (done) => {
-      mockAuthorizeRequest(up);
-      const scope = nock(REQ_OPTS.uri)
-                        .get('/?userProject=user-project-id')
-                        .replyWithFile(200, dawPath);
+      const scopes = [
+        mockAuthorizeRequest(),
+        nock(url).get(queryPath).replyWithFile(200, dawPath)
+      ];
       up.getRequestStream(REQ_OPTS, (requestStream: stream.Readable) => {
+        scopes[0].done();
         assert(requestStream);
         setImmediate(done);
         return through();
       });
     });
 
     it('should set the callback to a noop', (done) => {
-      mockAuthorizeRequest(up);
-      const scope =
-          nock(REQ_OPTS.uri).get('/?userProject=user-project-id').reply(200);
+      const scope = mockAuthorizeRequest();
+      // This scope is not checked, but required.  It is completed after the
+      // request stream is returned.
+      nock(url).get(queryPath).reply(200);
       up.getRequestStream(
           REQ_OPTS, (requestStream: stream.Readable&{callback: Function}) => {
+            scope.done();
             assert.strictEqual(
                 requestStream.callback.toString(), 'function () { }');
             done();
           });
     });
 
     it('should destroy the stream if there was an error', (done) => {
-      mockAuthorizeRequest(up);
-      const scope =
-          nock(REQ_OPTS.uri).get('/?userProject=user-project-id').reply(200);
+      const scopes =
+          [mockAuthorizeRequest(), nock(url).get(queryPath).reply(200)];
       up.getRequestStream(REQ_OPTS, (requestStream: stream.Readable) => {
-        const error = new Error(':(');
+        // This scope is not checked, but required.  It is completed after the
```

Comment:
Unclear what `This` refers to. The next line is `scopes[0]`, but I don't think the comment is about that scope.

Reply:
Apologies. Fixed to make it more clear.